lms/static/sass/features/_course-experience.scss

```diff
@@ -1,3 +1,66 @@
+// Course call to action message
+.course-message {
+  .message-author {
+    display: inline-block;
+    width: $baseline*7/2;
```

Comment:
This seems a rather contrived calculation. Is this size based upon the recommended image size? I'd be tempted to have a variable for this that isn't based upon ``$baseline``.